terraform/modules/testing/security_groups.tf

```diff
@@ -34,3 +34,46 @@ resource "aws_security_group_rule" "ingress_https_vpc_endpoints_from_batch" {
   type                     = "ingress"
   source_security_group_id = aws_security_group.batch_job_sg.id
 }
+
+resource "aws_security_group_rule" "ingress_to_push_gateway" {
+  from_port                = 9091
+  to_port                  = 9091
+  protocol                 = "tcp"
+  type                     = "ingress"
+  security_group_id        = var.push_host_sg
+  source_security_group_id = aws_security_group.sg_for_metric_lambda.id
+  description              = "Allow push access from Metrics Lambda"
+}
+
+// Security group for metrics lambdas that are placed in VPCs
+resource "aws_security_group" "sg_for_metric_lambda" {
```

Comment:
Should the security groups not be limited to specific port/tcp and the security group that it is going to or coming from. The name probably doesn't need to include the "sg_for" part either

Reply:
More restrictive egress rules added